src/erlcloud_sns.erl

```diff
@@ -500,11 +502,11 @@ set_topic_attributes(AttributeName, AttributeValue, TopicArn, Config)
             {"TopicArn", TopicArn}]).
 
 
--spec(subscribe/3 :: (string(), sns_subscribe_protocol_type(), string()) -> Arn::string()).
+-spec(subscribe/3 :: (parameter(), sns_subscribe_protocol_type(), parameter()) -> Arn::string()).
```

Comment:
Line 505 is too long: -spec(subscribe/3 :: (parameter(), sns_subscribe_protocol_type(), parameter()) -> Arn::string())..